master/getting-started/docker/tutorials/basic.md

```diff
@@ -91,37 +84,15 @@ possible to have an IPv6-only network, and (unlike the IPv4 behavior) it is
 necessary to specify a subnet from which to assign addresses.
 
 When running in a cloud environment we need to also set `ipip` and
-`nat-outgoing` options. If using the Calico IPAM driver `calico`, the
-`ipip` and `nat-outgoing` options are configured on the Calico IP Pool.
-
-#### The default IPAM driver.
-
-When a network uses the Docker default IPAM driver, a new container on the
-network is allocated an IP address from the network's CIDR. The Calico plugin
-assigns this IP address to the Calico interface on the container. In addition to
-this, the container connects to the host's Docker gateway bridge over a separate
-interface on the container. All non-network traffic (i.e. destinations outside
-the CIDR) is routed via the Docker gateway bridge and may not be subjected to the
-Calico policy.
-
-Since the container is connected to the Docker gateway bridge, it can utilize
-Docker's port mapping feature.  However, it is important to note that using
-Docker's port-mapping feature is not secured by Calico policy since the packets
-are routed via the Docker bridge, rather than through the Calico interfaces.
-(For more information on port-forwarding with Calico, check out the [Expose
-Ports to Internet guide]({{site.baseurl}}/{{page.version}}/usage/external-connectivity).)
-
-When running in a cloud environment we need to also set `ipip` and
-`nat-outgoing` options. If using the default IPAM driver, `ipip` and
-`nat-outgoing` are specified as options on the `network create`.
+`nat-outgoing` options in `ipPool` resource spec in the `yaml` or `json` config file.
 
 ### 3.2 Create the network
 
-So, with the IPAM driver selected, we can start creating some networks.
+So, with the Calico IPAM driver configured, we can start creating some networks.
 
 We specify the Calico networking driver (`calico`) when creating the network,
-and optionally specify the Calico IPAM driver (`calico`) if you chose to use
-Calico IPAM.
+and specify the Calico IPAM driver (`calico-ipam`). Note: `--ipam-driver calico-ipam`
```

Comment:
I assume that's true is it that it's called calico-ipam?  If you're not sure, check with Tom.